libnavui-alert/src/main/java/com/mapbox/navigation/ui/alert/AlertProcessor.kt

```diff
@@ -2,11 +2,11 @@ package com.mapbox.navigation.ui.alert
 
 import com.mapbox.navigation.ui.base.model.AlertState
 
-typealias SetAlertText<T1, T2> = (input: T1) -> T2
+internal typealias SetAlertText<T1, T2> = (input: T1) -> T2
 
-typealias SetDismissDuration<T1, T2> = (input: T1) -> T2
+internal typealias SetDismissDuration<T1, T2> = (input: T1) -> T2
```

Comment:
Same as above re: `private` right?

Reply:
nope, that's right. Has to be `internal`